### ISSUE TITLE
Sort out mayavi build issue

### ISSUE THREAD

**Issue by mrakitin** [NSLS-II/lightsource2-recipes#625]
It was attempted to build via https://github.com/NSLS-II/lightsource2-recipes/pull/604, but we eventually gave up in https://github.com/NSLS-II/lightsource2-recipes/pull/612.

**Comment by mrakitin** [comment]
No, it still fails with:
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"python[version='>=3.6,<3.7.0a0']", 'python=3.7'}
```

**Comment by CJ-Wright** [comment]
where is that coming from?

**Comment by mrakitin** [comment]
A more detailed traceback:
```py
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
Adding in variants from internal_defaults
Adding in variants from argument_variants
INFO:conda_build.variants:Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from argument_variants
Collecting package metadata: ...working... done
Solving environment: ...working... done
Collecting package metadata: ...working... done
Solving environment: ...working... failed
Traceback (most recent call last):
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda_build/environ.py", line 753, in get_install_actions
    actions = install_actions(prefix, index, specs, force=True)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/plan.py", line 473, in install_actions
    txn = solver.solve_for_transaction(prune=prune, ignore_pinned=not pinned)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/core/solve.py", line 107, in solve_for_transaction
    force_remove, force_reinstall)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/core/solve.py", line 145, in solve_for_diff
    force_remove)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/core/solve.py", line 242, in solve_final_state
    ssc = self._run_sat(ssc)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/core/solve.py", line 500, in _run_sat
    ssc.solution_precs = ssc.r.solve(tuple(final_environment_specs))
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/common/io.py", line 85, in decorated
    return f(*args, **kwds)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/resolve.py", line 1047, in solve
    self.find_conflicts(specs)
  File "/conda/tmp_envs/55fc582f7dd397d3a5f0/lib/python3.7/site-packages/conda/resolve.py", line 316, in find_conflicts
    raise UnsatisfiableError(bad_deps)
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - python=3.7
  - vtk=8.1 -> python[version='>=3.6,<3.7.0a0']
Use "conda search <package> --info" to see the dependencies for each package.
```

**Comment by CJ-Wright** [comment]
Interesting, the defaults py37 vtk build is 8.2.

**Comment by CJ-Wright** [comment]
It seems that the newest mayavi recipe from CF supports vtk 8.2

**Comment by mrakitin** [comment]
Cool! Will look into updating it later.

**Comment by mrakitin** [comment]
It seems to be resolved via #627.
